pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -3,83 +3,60 @@
 #
 #   Typically in /etc/wavefront/wavefront-proxy/wavefront.conf
 #
-##############################################################################
-# The server should be either the primary Wavefront cloud server, or your custom VPC address.
-#   This will be provided to you by Wavefront.
-#
+########################################################################################################################
+# Wavefront API endpoint URL. This will be provided to you by Wavefront, usually the same as URL of your Wavefront
+# instance, with "/api/" suffix.
 server=https://try.wavefront.com/api/
 
 # The hostname will be used to identify the internal proxy statistics around point rates, JVM info, etc.
-#  We strongly recommend setting this to a name that is unique among your entire infrastructure,
-#   possibly including the datacenter information, etc. This hostname does not need to correspond to
-#   any actual hostname or DNS entry; it's merely a string that we pass with the internal stats.
-#
+# We strongly recommend setting this to a name that is unique among your entire infrastructure, to make this
+# proxy easy to identify. This hostname does not need to correspond to any actual hostname or DNS entry; it's merely
+# string that we pass with the internal stats and ~proxy.* metrics.
 #hostname=my.proxy.host.com
 
-# The Token is any valid API Token for your account, which can be generated from the gear icon
-#   at the top right of the Wavefront site, under 'Settings'. Paste that hexadecimal token
-#   after the '=' below, and the proxy will automatically generate a machine-specific UUID and
-#   self-register.
-#
-#token=XXX
-
-## Set to true when running proxy inside containers or when the proxy is frequently restarted on a new infrastructure.
-## When true, terminated proxies will be automatically removed from the UI after 24 hours of inactivity.
-#ephemeral=false
+# The Token is any valid API Token for your account that has "Proxy Management" permissions. This token can be
+# by clicking on the gear icon at the top right of the Wavefront site -> click on your email -> click on "API access".
+#token=
```

Comment:
The token is any valid API token for an account that has **Proxy Management** permissions. To get to the token: 
1. Click the gear icon at the top right in the Wavefront UI. 
2. Click your account name (usually your email)
3. Click **API access**.